puppet/spec/classes/profiles_web_spec.rb

```diff
@@ -4,12 +4,11 @@
   on_supported_os.each do |os, os_facts|
     context "on #{os}" do
       let(:facts) { os_facts }
-      let(:params) { {setup_receiver: false} }
```

Comment:
Should we drop the whole parameter? It was only used for testing.

Reply:
If it was only for testing, yes!